src/GovUk.Education.ExploreEducationStatistics.Data.Model/Repository/GeoJsonRepository.cs

```diff
@@ -34,31 +34,31 @@ public class GeoJsonRepository : IGeoJsonRepository
                 .ToDictionary(geoJson => geoJson.Code);
         }
 
-        private IEnumerable<GeoJson> FindAndCache(long boundaryLevelId, IEnumerable<string> codes)
+        private IEnumerable<BoundaryData> FindAndCache(long boundaryLevelId, IEnumerable<string> codes)
         {
-            var geoJsonList = _context.GeoJson.Where(geoJson =>
-                geoJson.BoundaryLevelId == boundaryLevelId &&
+            var geoJsonList = _context.BoundaryData.Where(geoJson =>
+                geoJson.BoundaryLevel.Id == boundaryLevelId &&
                 codes.Contains(geoJson.Code)).ToList();
```

Comment:
I think this query on the `BoundaryData` table would benefit from having an index on the `Code` column which doesn't exist in the configuration you've applied to it so far.